### DISCUSSION TITLE
Search by Element ID

### DISCUSSION THREAD

**Discussion by rjhewitt3** [craftcms/cms#8947] [Ideas]
There was an issue running in Commerce that mentioned this, but we need a way to search based on Element ID, as @lukeholder pointed out was the real issue. For example, in Commerce, we need to be able to search orders based on their ID and under Users as well, and it would be helpful in all other contexts as well (ex Entries). These numbers are used elsewhere for reference irrespective of any custom Referencing format we've created, so it's necessary in order to cross-reference things.

For more information on this issue, and how its' been described by others, please see the closed issue in Commerce: https://github.com/craftcms/commerce/issues/1077

**Comment by brandonkelly** [comment]
We will be making this possible in Craft 4. In the meantime, as of Craft 3.5.13 you can go to `/admin/edit/X` and you will be redirected to the element’s order page, whether it’s a user, order, product, entry, asset, or any other element type that has edit pages in the CP. (Replace `X` with the element’s ID or UID.)
